### PR TITLE
refactor controllers, change module prefix to `chatspaces`

### PR DESCRIPTION
**Pull request by btford**
I'm still working on this PR, but thought it'd be good to get some early feedback.

I thought it was confusing that the modules were prefixed with `chatspace` when all of the docs refer to the app as `chatspaces,` so I updated the module names accordingly. I also moved some of the controllers out of the single `controllers.js` file as the start of some larger refactoring.

What do you think of this direction?

### CODE REVIEW

**Comment by operatorjen** [comment]
looks good so far! added some comments inside (mainly side notes on things i wasn't sure about)

File: public/javascripts/controllers.js [hunk 1]
```diff
@@ -1,476 +1,324 @@
...
-      }
+    }
+  };
 
-      if (!$scope.posting) {
-        $scope.posting = true;
+  $scope.cancelCamera = function () {
+    $scope.showCamera = false;
+    $scope.showFollowing = false;
+    $('#video-preview').empty();
```
Comment (operatorjen) [inline public/javascripts/controllers.js:226]: wasn't sure how to empty a dom element in angular in some graceful way - do i have to assign a scope to the child elements to do the same thing? (i actually have a todo in the factories.js https://github.com/mozilla/chatspaces/blob/master/public/javascripts/factories.js#L145-L146 relating to this)

**Comment by operatorjen** [comment]
closing since i refactored a bit!

**Comment by btford** [comment]
woop!

**Pull request closed without merging**